recipes/serial-monitor-rust/recipe.yaml

```diff
@@ -15,41 +15,29 @@ build:
   script:
     - if: osx
       then:
-        - clang --version
+        - export PATH="/usr/bin:$PATH"  # Ensure that system Clang is used, not a potentially conflicting Homebrew version
         - export MACOSX_DEPLOYMENT_TARGET=10.15  # or the correct macOS version
         - export CC=/usr/bin/clang
         - export CXX=/usr/bin/clang++
         - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
-        - cargo install --locked --bins --root ${PREFIX} --path .
+        - cargo install --bins --root ${PREFIX} --path .
     - if: linux
       then:
         - conda install -c conda-forge \
           libclang libgtk-3 libxcb libxkbcommon openssl libudev -y
         - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
-        - cargo install --locked --bins --root ${PREFIX} --path .
+        - cargo install --bins --root ${PREFIX} --path .
     - if: win
       then:
         - cargo-bundle-licenses --format yaml --output ${SRC_DIR}/THIRDPARTY.yml
-        - cargo install --locked --bins --root ${PREFIX} --path .
+        - cargo install --bins --root ${PREFIX} --path .
 
 requirements:
   host:
     - ${{ compiler('rust') }}
```

Comment:
if you need c or c++ compilers, add `${{ compiler('c') }}` or `${{ compiler('cxx') }}`.